components/automate-ui/src/app/pages/policy/add-members/policy-add-members.spec.ts

```diff
@@ -0,0 +1,98 @@
+    let fixture: ComponentFixture<PolicyAddMembersComponent>;
+
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                MockComponent({ selector: 'chef-page' }),
+                MockComponent({ selector: 'chef-modal', inputs: ['visible'] }),
+                MockComponent({ selector: 'chef-form-field' }),
+                MockComponent({ selector: 'chef-error'}),
+                MockComponent({ selector: 'chef-button', inputs: ['disabled'] }),
+                MockComponent({ selector: 'chef-input'}),
+                MockComponent({ selector: 'chef-table' }),
+                MockComponent({ selector: 'chef-thead' }),
+                MockComponent({ selector: 'chef-tr' }),
+                MockComponent({ selector: 'chef-th' }),
+                MockComponent({ selector: 'chef-tbody' }),
+                MockComponent({ selector: 'chef-td' }),
+                MockComponent({ selector: 'chef-checkbox', inputs: ['checked'] }),
+                MockComponent({ selector: 'a', inputs: ['routerLink'] }),
+                PolicyAddMembersComponent
+            ],
+            imports: [
+                ChefPipesModule,
+                ReactiveFormsModule,
+                RouterTestingModule,
+                StoreModule.forRoot({
+                    router: () => ({
+                        state: {
+                            url: '/settings/policies/editor-access/add-members',
+                            params: {}
+                        }
+                    })
+                }, { runtimeChecks })
+            ]
+        }).compileComponents();
+    }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(PolicyAddMembersComponent);
+        component = fixture.componentInstance;
+        component.expressionForm = new FormBuilder().group({
+            type: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]],
+            identity: [''],
+            name: ['']
+        });
+        fixture.detectChanges();
+    });
+
+    it('should be created', () => {
+        expect(component).toBeTruthy();
+    });
+
+    describe('showInputs', () => {
+
+        it('should return false when inputName is not identity or name', () => {
+            expect(component.showInputs('blank')).toBe(false);
+        });
+
+        using([
+            ['user', true, 'identity and when typeValue equals user', 'identity'],
+            ['team', true, 'identity and when typeValue equals team', 'identity'],
+            ['*', false, 'identity and typeValue does not equal team or user', 'identity'],
+            ['token', true, 'name and when typeValue equals token', 'name'],
+            ['team', false, 'name + typeValue is not token + identityValue does not exist', 'name']
+        ], function (type: string, outcome: boolean, description: string, input: string) {
+            it(`should return ${outcome} when inputName is ${description}`, () => {
```

Comment:
```suggestion
            it(`returns ${outcome} when inputName is ${input} ${description}`, () => {
```
... with supporting changes to description.